src/commons/services/utils.js

```diff
@@ -235,8 +235,8 @@
                 getEventListRequest,
                 request;
 
-            //url = "/api/events.json";
-            url = config.baseUrl + "clusters/" + clusterId + "/notifications";
+            url = "/api/events.json";
+            //url = config.baseUrl + "clusters/" + clusterId + "/notifications";
```

Comment:
I thought this change only helps with testing. Should it be here?

Reply:
@shirshendu Yes. I will remove it.